src/main/webapp/add-account.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Add File To Configuration</title>
+    <link rel="stylesheet" href="assets/add-account.css">
+    <link rel="stylesheet" href="assets/common.css">
+    <script src="scripts/appscript.js"></script>
+  </head>
+  <body onload="javascript:populateCustomerList()">
+    <div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="readfile.html">Read Configuration</a>
+        <a href="createfile.html">Create Configuration</a>        
+        <a href="editfile.html">Edit Configuration</a>
+        <a href="deletefile.html">Delete Configuration</a>
+		  </div>
+    </div>
+    <h1>Billing Configuration Sync Tool</h1></br>
+    <h2>Add Account</h2>
+    <div id="edit-account">
+      <form id="config-search" onsubmit="buildEditForm(); return false;">
```

Comment:
what is the purpose of returning false here?

Reply:
Returning false allows for alerts to be visible long enough for users to click out of them. Without it, it pops up and disappears immediately after.